tests/python/gaia-ui-tests/gaiatest/apps/settings/app.py

```diff
@@ -144,97 +178,123 @@ def language_menu_item_description(self):
-        self._tap_menu_item(self._findmydevice_locator)
-        return FindMyDevice(self.marionette)
+    def open_media_storage_settings(self):
+        return self._open_subpage_settings(self._media_storage_menu_item_locator, 'media_storage', 'MediaStorage')
+
+    def open_application_storage_settings(self):
+        return self._open_subpage_settings(self._application_storage_menu_item_locator)
 
     def open_device_info_settings(self):
-        from gaiatest.apps.settings.regions.device_info import DeviceInfo
-        self._tap_menu_item(self._device_info_menu_item_locator)
-        return DeviceInfo(self.marionette)
+        return self._open_subpage_settings(self._device_info_menu_item_locator, 'device_info', 'DeviceInfo')
+
+    def open_downloads_settings(self):
+        return self._open_subpage_settings(self._downloads_menu_item_locator)
 
     def open_battery_settings(self):
-        from gaiatest.apps.settings.regions.battery import Battery
-        self._tap_menu_item(self._battery_menu_item_locator)
-        return Battery(self.marionette)
+        return self._open_subpage_settings(self._battery_menu_item_locator, 'battery', 'Battery')
 
-    def open_sim_manager_settings(self):
-        from gaiatest.apps.settings.regions.sim_manager import SimManager
-        self._tap_menu_item(self._sim_manager_menu_item_locator)
-        return SimManager(self.marionette)
+    def a11y_open_accessibility_settings(self):
+        return self._open_subpage_settings(self._accessibility_menu_item_locator, 'accessibility', 'Accessibility')
 
-    def open_homescreen_settings(self):
-        from gaiatest.apps.settings.regions.homescreen_settings import HomescreenSettings
-        self._tap_menu_item(self._homescreen_menu_item_locator)
-        return HomescreenSettings(self.marionette)
+    def open_developer_settings(self):
+        return self._open_subpage_settings(self._developer_menu_item_locator)
 
-    def open_browsing_privacy_settings(self):
-        from gaiatest.apps.settings.regions.browsing_privacy import BrowsingPrivacy
-        self._tap_menu_item(self._browsing_privacy_item_locator)
-        return BrowsingPrivacy(self.marionette)
+    def open_improve_settings(self):
+        return self._open_subpage_settings(self._improve_menu_item_locator)
+
+    def open_help_settings(self):
+        return self._open_subpage_settings(self._help_menu_item_locator)
+
+    def _open_subpage_settings(self, locator, file_name=None, class_name=None):
+        self._tap_menu_item(locator)
+        class_object = self._get_class_by_name(file_name, class_name)
+        if class_object is not None:
+            return class_object(self.marionette)
+
+    def _get_class_by_name(self, file_name, class_name):
+        if file_name is not None:
+            package_path = 'gaiatest.apps.settings.regions.{}'.format(file_name.lower())
+            mod = __import__(package_path, fromlist = [class_name])
+            class_object = getattr(mod, class_name)
```

Comment:
I'd rename it to `ClassObject` because we're retrieving the class itself which will be used as a constructor at [line 290](https://github.com/mozilla-b2g/gaia/pull/30847/files#diff-716c59d65bea2e1456305b9fc9d6caacR290).